llvm/lib/Support/VirtualFileSystem.cpp

```diff
@@ -302,6 +305,17 @@ class RealFileSystem : public FileSystem {
     return Storage;
   }
 
+  ErrorOr<std::unique_ptr<File>> openFileForRead(const Twine &Name,
```

Comment:
it was quite confusing to see this as an overload, can you call it `openFileForReadWithFlags` ?

Reply:
Thanks, this is done